DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -31,13 +31,30 @@ public void Get_section_content_should_return_section_content()
 
             // Then
             var expectedSectionContent = new SectionContent(
-                "Erin Test 01",
                 "Office 2013 Essentials for the Workplace",
+                "Erin Test 01",
                 "Working with Microsoft Office applications",
                 0,
                 28,
                 true,
-                0
+                0,
+                0,
+                13,
+                "https://www.dls.nhs.uk/CMS/CMSContent/Course120/Diagnostic/01-Diag-Working-with-Microsoft-Office-applications/itspplayer.html",
+                "https://www.dls.nhs.uk/CMS/CMSContent/Course120/PLAssess/01-PLA-Working-with-Microsoft-Office-applications/itspplayer.html",
+                1,
+                1,
+                true,
+                true
+            );
+            expectedSectionContent.Tutorials.AddRange(
+                new[]
+                {
+                    new SectionTutorial("Introduction to applications", 0, "Not started", 0, 17),
+                    new SectionTutorial("Common screen elements", 0, "Not started", 0, 11),
+                    new SectionTutorial("Using ribbon tabs", 0, "Not started", 0, 6),
+                    new SectionTutorial("Getting help", 0, "Not started", 0, 11)
+                }
```

Comment:
Do we have a test for getting the section content if there's no progress, i.e. the user isn't enrolled, like we do for the initial menu content? 🙂

Reply:
Does the test for an unenrolled candidate count? As a candidate that isn't enrolled won't have a progressId for it?

Reply:
Yes, that's what I meant. Couldn't see it so just wanted to make sure we had it, as that should make sure that we have correct default values for when there's no progress 🙂